docs/source/index.rst

```diff
@@ -1,119 +1,135 @@
-Dissect
-=======
+Welcome to Dissect!
+===================
 
-Welcome to the official documentation of Dissect!
+Effortlessly extract and investigate **forensic artefacts** from any source with **Dissect**.
+With Dissect, you can go from intake call to patient zero in a matter of hours,
+even in infrastructures with thousands of systems.
 
-Dissect is a collection of Python libraries and tools to facilitate enterprise-scale incident response and forensics.
-It supports you, the analyst, from the moment of acquisition of artefacts, to normalisation and processing.
 
-Dissect frees you from limitations by data formats and platforms and takes away concerns about how to access
-investigation data. You can focus on performing analysis, developing complex analysis plugins or performing
-research. You know, the cool stuff that we brag about on birthday parties. With Dissect, you can go from intake call
-to patient zero in a matter of hours, even in infrastructures with thousands of systems.
+Quick Demo
+----------
 
-With Dissect, beginner and intermediate analysts get direct access to a large collection of artefact parsers and
-plugins that work quickly and easily on a large range of evidence formats. More advanced analysts with scripting
-experience can also leverage Dissect`s full capabilities by creating new tools and plugins using the various Dissect
-APIs and parsers.
+In just a couple of seconds you can query a **forensic image** or even open a shell!
 
+.. image:: /images/demo.gif
 
-.. note ::
-    Read more about what Dissect is and how it works at :doc:`/overview/index` or check out what others
-    have written about Dissect in :doc:`/resources/dissect-in-action`.
 
+Usage Example
+-------------
+
+One of the most prominent tools that Dissect offers is called *target-query*
+With this simple one-liner you can extract all user accounts from an image: 
 
-Getting Started
----------------
+.. code-block:: console
 
-The easiest way to get started is to install the latest version of all Dissect projects from
-`PyPI <https://pypi.python.org/pypi/dissect>`_:
+    $ target-query pc.img -f users
+    
+    <windows/user ... name='systemprofile' ...>
+    <windows/user ... name='LocalService' ...>
+    <windows/user ... name='NetworkService' ...>
+    <windows/user ... name='Mr. Evil' ...>
+    
+To see what other useful artefacts you can query use ``-l``:
 
 .. code-block:: console
 
-    $ pip install dissect
+    $ target-query pc.img -l
 
-It is recommended that you use a `virtual environment <https://docs.python.org/3/tutorial/venv.html>`_.
+This will list all Dissect functions and artefacts that are available for this image.
```

Comment:
```suggestion
This will list all Dissect functions and artefacts that are available for **this** image.
```

Reply:
Unfortunately that change would interfere with SEO. A better way would be to apply a style, but I don't have direct control over HTML.